arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -3926,7 +3829,13 @@ static void wifi_coex_deinit(void)
 
 static int wifi_coex_enable(void)
 {
+#if defined(CONFIG_ESP32_WIFI_BT_COEXIST)
+  int ret;
+  ret = coex_enable();
+  return ret;
```

Comment:
```suggestion
  return coex_enable();
```
nit: This may be simplified